### ISSUE TITLE
autofarm bugs on chest and shulker boxes 

### ISSUE THREAD

**Issue by alief03** [Wurst-Imperium/Wurst7#1081]
### Pre-reporting checklist

- [X] I have searched existing issues and didn't find any previous reports of this bug.
- [X] This bug still happens in the latest version of Wurst. I've made sure my installation is up to date.
- [ ] I have tried removing all other mods and the bug still happens.

### Description

when i enable autofarm it runs close to chest or shulker box, Suddenly chest/shulker box opens (with no intention of opening) and cannot be closed 

### Repro steps

1. Enable Autofarm
2. walk closer to chest/shulker box
3. suddenly chest/shulker box opens
4. cannot be closed, unless reopening the game

### Wurst Client version

7.46

### Minecraft version

1.21

**Comment by Alexander01998** [comment]
Yeah, this is a somewhat known issue that affects many different hacks in Wurst. The problem is that there's no reliable way to detect if a block is "interactive" (meaning if right-clicking it does something like opening a chest or shulker box) without actually just right-cliking that block and seeing what happens.

I say it's "somewhat known" because I'm not sure why AutoFarm would try to right-click on anything other than plants and farmland in the first place. Maybe its aim is slightly off with some specific crop or growth stage?

Implementing #1075 should somewhat mitigate the issue, as then we'll have at least a basic filter for chests and shulker boxes in every hack that right-clicks on stuff. But if it is an aim issue with AutoFarm, where it's trying to click on a crop but ends up pointing at the chest instead, then this would probably still happen even with that fix.

Let me know if you figure out a specific farm setup that can reproduce the bug.

**Comment by HackerXGF** [comment]
Wurst for Minecraft 1.21.3 doesn't start for me, all previous versions work really, and the new one, when I start it, it shows me the message the game has crashed, an unexpected problem has occurred. We apologize for the inconvenience. Exit code 1.

**Comment by Alexander01998** [comment]
@HackerXGF Please don't hijack someone else's issue or complain about crashes without including the necessary troubleshooting data from your crash report file. This kind of behavior is rude and will not fix whatever issue you're encountering.

**Comment by HackerXGF** [comment]
I wouldn't be writing this post if it was clearly written where to create a New Issue.

**Comment by Alexander01998** [comment]
Hey @HackerXGF. There seems to be a misunderstanding here. I am not the CEO of GitHub who gets to decide where the "New Issue" button goes. Whatever gripes you have with GitHub's UI, feel free to tell them in the [community discussions](https://github.com/orgs/community/discussions), but please leave me out of it as I can't do anything about that.

**Comment by tolland** [comment]
Yeah, I had this problem with my ButtonAuraHack, the net.wurstclient.util.BlockPlacer#getBlockPlacingParams is returning blocks to place against which have interaction menus on right click.

If you run the gametest in this branch: https://github.com/Wurst-Imperium/Wurst7/compare/master...tolland:Wurst7:block-place-test?expand=1

the North placed chest is returned for the getBlockPlacingParams and it gets stuck in the menu:

<img width="863" height="502" alt="Image" src="https://github.com/user-attachments/assets/6f26f64b-ea3d-4ae4-b718-4e4b4608aa9c" />

Originally, I did the static check of the block isinteractive and then sneaked to place: 

https://github.com/tolland/Wurst7/blob/1.21.8-develop/src/main/java/net/wurstclient/hacks/ButtonAuraHack.java#L354

that does work, but causing annoying bobbing, so I think probably fix in the upstream method
